src/libcore/hash/sip.rs

```diff
@@ -30,22 +48,30 @@ use super::Hasher;
 /// Although the SipHash algorithm is considered to be generally strong,
 /// it is not intended for cryptographic purposes. As such, all
 /// cryptographic uses of this implementation are _strongly discouraged_.
-#[derive(Debug)]
 #[stable(feature = "rust1", since = "1.0.0")]
-pub struct SipHasher {
+pub type SipHasher = SipHash24;
```

Comment:
We should probably make this an opaque tuple struct to allow us flexibility in the future to change what it corresponds to.